client/components/Item/RelatedItems.vue

```diff
@@ -109,12 +109,19 @@ export default Vue.extend({
     async refreshItems(): Promise<void> {
       this.loading = true;
 
+      const options = {
+        itemId: this.item.Id,
+        userId: this.$auth.user?.Id,
+        limit: this.vertical ? 5 : 12,
+        excludeArtistIds: [] as string[]
+      };
+
+      for (const albumArtist of this.item.AlbumArtists) {
+        options.excludeArtistIds.push(albumArtist.Id);
```

Comment:
If this is to be done, it should be on the server-side.

Doing it on the client hampers consistency across the various clients. We should strive to return the same results everywhere, not to manipulated them all the time to get what we want.

Reply:
I would be fine adding this with a `TODO` or `SERVER` comment. We definitely want to do things server-side when possible, but for smaller items it can be easier to implement within clients until the server has support, especially since we are striving towards a more "stable" server API in the future.

Reply:
This would be very easy to do on the server though, provided we're okay with slightly changing what is returned from it (It won't break clients, just deprecate a field and return different results).